src/main/java/org/folio/edge/orders/ValidateHandler.java

```diff
@@ -26,7 +26,22 @@ protected void handle(RoutingContext ctx) {
         (client, params) -> {
           PurchasingSystems ps = PurchasingSystems.fromValue(params.get(PARAM_TYPE));
           logger.info("Request is from purchasing system: " + ps.toString());
-          ((OrdersOkapiClient) client).validate(
+          ((OrdersOkapiClient) client).validate(HttpMethod.GET,
```

Comment:
`ctx.request().method()` ?  See comments below in `handlePost(...)`

Reply:
Done